tests/ExampleApps/HeadlessLibAndApp/Application/CMakeLists.txt

```
@@ -3,7 +3,16 @@ if(APPLE)
     cmake_policy(SET CMP0042 NEW)
 endif(APPLE)
 
-project(HeadlessDemo VERSION 1.0.0)
+if(IN_LIBHEADLESS_BUILD_TREE)
+    project(HeadlessDemo VERSION ${RadiumHeadless_VERSION})
+else()
+    project(HeadlessDemo VERSION 1.0.0)
+endif()
```

```suggestion
project(HeadlessDemo VERSION 1.0.0)
```
Well I think we should not have different version depending on how apps or libs are compiled.
Let's keep this 1.0.0 version (same for other apps/lib) and keep Radium_VERSION for bundled lib  (core, engine, ...)